.idea/gradle.xml

```diff
@@ -7,7 +7,6 @@
         <option name="testRunner" value="PLATFORM" />
```

Comment:
it's better to exclude entire `.idea` directory from git (write rule in .gitignore file)